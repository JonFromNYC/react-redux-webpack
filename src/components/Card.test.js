import { shallow } from 'enzyme';
import React from 'react';
import Card from './Card.component';

// const wrapper = shallow(<Foo />);
it('expect to render a Card component', ()=>{
    expect(shallow(<Card />)).toMatchSnapshot();
});