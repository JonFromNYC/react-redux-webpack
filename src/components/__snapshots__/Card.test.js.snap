// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render a Card component 1`] = `ShallowWrapper {}`;
